static/js/containers/LearnerSearchPage_test.js

```diff
@@ -104,7 +100,38 @@ describe('LearnerSearchPage', function () {
         TestUtils.Simulate.click(saveButton);
       }).then(() => {
         assert.isFalse(helper.store.getState().ui.emailDialogVisibility);
-        assert.isTrue(sendSearchResultMail.calledWith("subject", "body", {abc: 123}));
+        assert.isTrue(sendSearchResultMail.calledWith("subject", "body", searchkit.buildQuery().query));
+      });
+    });
+  });
+
+  it('filters names if there is text in the textbox', () => {
+    return renderComponent('/learners').then(([wrapper]) => {
+      return new Promise(resolve => {
+        wrapper.find("SearchBox").find('input[type="text"]').props().onInput({
+          target: {
+            value: 'xyz'
+          }
+        });
```

Comment:
The timeout is necessary because searchkit waits a little bit before firing off a request after a person types in text